### DISCUSSION TITLE
Tree container interface

### DISCUSSION THREAD

**Discussion by Kevin-Delnoije** [stdgraph/graph-v2#56] [Ideas]
Hey,

I was wondering what are the current plans for a tree container? There are several algorithms that result in a tree,so some kind of container is helpful there.  
Besides the algorithms in this library a lot of domains use tree data structures, for example k-d tree and quad-tree  in computer vision.
This might be out of scope for this library but if it is possible to implement these kind of data structures with a consistent interface that would be worth exploring

**Comment by pratzl** [comment]
I've heard that suggestion before, there's no intention to add a tree container interface. We're keeping busy with graph-related things.
